dali/c_api/c_api_test.cc

```diff
@@ -42,7 +42,7 @@ constexpr bool pipelined = true;
 constexpr int prefetch_queue_depth = 2;
 constexpr bool async = true;
 constexpr float output_size = 20.f;
-constexpr cudaStream_t cuda_stream = 0;
+const cudaStream_t cuda_stream = cudaStreamLegacy;
```

Comment:
The compiler protests that `cudaStreamLegacy` defined as `((cudaStream_t)1)` is not a constant expression, since it casts an integer to a pointer.